docs/run/primitives.mdx

```diff
@@ -80,8 +80,8 @@ estimator.run([circuit1, circuit2, ...],[observable1, observable2, ...],[param_v
 
 #### Estimator V2
 
-* Takes one parameter: PUBs in the format (`<circuit>`, `<observables>`, `<parameter values>`, `<precision>`)
-* Numpy [broadcasting rules](https://numpy.org/doc/stable/user/basics.broadcasting.html) are used when combining observables and parameter values.
+* Takes one parameter: PUBs in the format (`<circuit>`, `<observables>`, `<parameter values>`, `<precision>`) (Note that there is only one circuit and optional `precision` value per PUB, but `parameter values` and `observables` can be lists or a single value.)
```

Comment:
parameter values are also optional. Now that I'm reading this line, it has some other problems. Estimator V2 `run()` actually takes `pubs` (which is an array of PUB) _and_ `precision` (same goes to Sampler which takes `pubs` and `shots`). 

Would it be clearer if we just say
"
The `run()` method takes an array of PUBs. Each PUB is  in the format (`<single circuit>`, `<one or more observables>`, `<optional one or more parameter values>`, `<optional precision>`)
"